deploy/manifests/scc.yaml

```diff
@@ -0,0 +1,37 @@
+# Create SCC for Gateway resources
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: nginx-ngf-admin
+allowPrivilegedContainer: false
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPorts: false
+readOnlyRootFilesystem: true
+runAsUser:
+  type: MustRunAsRange
```

Comment:
Why do we use Ranges for the user and group IDs instead of single values?

Reply:
[/apis/security.openshift.io/v1](https://docs.okd.io/3.10/rest_api/apis-security.openshift.io/v1.PodSecurityPolicyReview.html)

I will try playing around with group IDs using single values and using `MustRunAs` for `runAsUser` and listing multiple `uids` but based off of the documentation I linked above, the current super unintuitive way is what I think the docs point to

Reply:
For the group IDs I tried

```
fsGroup:
  type: MustRunAs
  value: 1001

and 

fsGroup:
  type: MustRunAs
  uid: 1001
```
and got this error 

```
provider nginx-ngf-admin: .spec.securityContext.fsGroup:
      Invalid value: []int64{1001}: 1001 is not an allowed group
```

For the runAsUser when I tried changing it to

```
runAsUser:
   type: MustRunAs
   uid: 101, 102
```

I got this error:

`Error from server: error when creating "scc.yaml": json: cannot unmarshal string into Go struct field RunAsUserStrategyOptions.runAsUser.uid of type int64`


Is there anything else you'd like me to try? My findings are in alignment with the documentation on what the fields are supposed to be, even though they are super unintuitive.

Reply:
I understand that the user ID has to be a range because we have two containers, but it seems like the group ID should be a single value. But, since that isn't working, I relent. Thanks for checking!